app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java

```diff
@@ -66,6 +67,13 @@ public Object instantiateItem(ViewGroup container, int position) {
             }
             ViewHolder holder = new ViewHolder(layout);
             layout.setTag(holder);
+
+            if(position == PAGE_FINAL){
```

Comment:
Could you please explain why this was included in this PR? I don't see anything wrong with the code per se, but it seems to address a separate issue.

Reply:
Sure @misaochan! Adding CDATA tag alone does not give the expected output. We have to use the setText(Html.fromHtml()) method. So, when the 'ViewGroup' object 'layout' holds the final layout page (which contains the string with CDATA tag as text), then only we have to set the text. That's why I have used an 'if' condition to know when 'layout' holds the targeted page.

Reply:
Make sense! Thanks. :) Happy to merge once this has been tested.

Reply:
Going back through `WelcomePagerAdapter` and didn't fully understand this - why is this enabled only for the beta version of the app?